pkg/controller/chaosengine/chaosengine_controller_test.go

```diff
@@ -78,6 +104,54 @@ func TestNewRunnerPodForCR(t *testing.T) {
 
 			isErr: false,
 		},
+		"Test Positive-3": {
+			engine: chaosTypes.EngineInfo{
+				Instance: &litmuschaosv1alpha1.ChaosEngine{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-runner",
+						Namespace: "test",
+					},
+					Spec: litmuschaosv1alpha1.ChaosEngineSpec{
+						ChaosServiceAccount: "fake-serviceAccount",
+						Monitoring:          false,
+						AnnotationCheck:     "false",
+						Components: litmuschaosv1alpha1.ComponentParams{
+							Runner: litmuschaosv1alpha1.RunnerInfo{
+								Image: "fake-runner-image",
+							},
+						},
+					},
+				},
+				AppUUID:        "fake_id",
+				AppExperiments: []string{"exp-1"},
+			},
+
+			isErr: false,
```

Comment:
Rather than just checking boolean error, we can map error strings with thier respective errors